5-WebApp-AuthZ/5-2-Groups/AppCreationScripts/sample.json

```diff
@@ -5,9 +5,23 @@
     "Client": "ASP.NET Core Web App",
     "Service": "Microsoft Graph",
     "RepositoryUrl": "microsoft-identity-platform-aspnetcore-webapp-tutorial",
-    "Endpoint": "AAD v2.0"
+    "Endpoint": "AAD v2.0",
+    "platform": "AspNetCore",
+    "Languages": [
+      "csharp"
+    ],
+    "Products": [
```

Comment:
>     "Products": [

Shouldn't be an array. Can you compare with the header here:  https://raw.githubusercontent.com/Azure-Samples/ms-identity-javascript-angular-tutorial/main/5-AccessControl/2-call-api-groups/README.md

Reply:
The `Products` property is registered as a list.

Is there something I'm missing?